src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -271,6 +272,8 @@ public function handleSignal(int $signal, int|false $previousExitCode = 0): int|
 
         $this->worker->stop();
 
+        $this->eventDispatcher?->dispatch(new WorkerStoppedEvent($this->worker));
```

Comment:
This doesn't look necessary to me. The event should already be dispatched at the end of the `Worker::run()` method.

Reply:
Are you sure that's still the case after the fix from #52080?

Reply:
i will check that